kotlin-test/src/test/kotlin/org/axonframework/extension/kotlin/test/FixtureExtensionsTest.kt

```diff
@@ -1,15 +1,52 @@
 package org.axonframework.extension.kotlin.test
 
-import org.axonframework.test.aggregate.AggregateTestFixture
 import kotlin.test.Test
 
 internal class FixtureExtensionsTest {
 
+    @Test
```

Comment:
Well, these tests are rather...boring of course. I know it's not entirely in the vein of what's required for each test case, but regardless I'd actually like to see an invocation of `whenever`/`when` _together_ with an `expect...` invocation. So taking it a step further through the entire fixture process really.

Reply:
I added some `expectNoEvents` invocations after `whenever`s because this is what the example aggregate do really. I am not entirely sure we should add these `expect...` invocations as they add noise to the tests and derive them from their actual purpose, i.e. testing the `whenever` invocation.

Reply:
Agreed, this looks good @Arnaud-J. Thanks for the adjustment :-)